src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs

```diff
@@ -2571,8 +2571,7 @@ private void RegisterZones()
                         zone.TerraformHandler = ctx.Resolve<TerraformHandler.Factory>().Invoke(zone);
                     }
 
-                    ctx.Resolve<EventListenerService>().AttachListener(new WeatherWatcher(zone));
-                    ctx.Resolve<EventListenerService>().AttachListener(new GameTimeEventProcessor(zone));
+                    ctx.Resolve<EventListenerService>().AttachListener(new EnvironEffectHandler(zone));
```

Comment:
`EnvironEffectHandler` can we change to `EnvironmentEffectHandler` or `EnvironmentalEffectHandler`

Reply:
+1